scripts/post_pr_comment.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os
+import sys
+import urllib2
+
+GITHUB_URL = 'https://api.github.com'
+
+def post_comment(pr_id, repo_slug, comment_body, user_token):
+	# Constructing the url
+	url = '{api_url}/repos/{slug}/issues/{number}/comments'.format(
+        api_url=GITHUB_URL, slug=repo_slug, number=pr_id)
+	req = urllib2.Request(url=url,
+                       data=json.dumps({'body': comment_body}))
+	req.add_header('Authorization', 'token ' + user_token)
+	# Making the request
+	f = urllib2.urlopen(req)
+	if f.getcode() != 201:
+		print f.read()
+
+
+if __name__ == '__main__':
```

Comment:
Considering we are invoking this through the interpreter, it shouldn't be necessary -- 

```python
import sys

print sys.stdin.read().strip()
```

And calling it as `python test.py < test.py` prints itself without the need of the wrapping block.

If we wanted to future-proof this, I'd put the block _within_ the if-name-main into a function that can be exported, and then within the block just call the function.